config/default.schema.json

```diff
@@ -141,7 +165,7 @@
 		"JWT_TIMEOUT_SECONDS": {
 			"type": "integer",
 			"default": 7200,
-			"description": "After this many seconds of inactivity, a user gets logged out automatically. If no REDIS_URI is set, this value is ignored."
+			"description": "After this many seconds of inactivity, a user gets logged out automatically. If no SESSION_VALKEY_URI is set, this value is ignored."
```

Comment:
If no SESSION_VALKEY_URI is set, this value is ignored --> stimmt so eigentlich nicht mehr. 
Das gleiche in Zeiel 178

Reply:
Done